colab/colab_utils.py

```diff
@@ -0,0 +1,1048 @@
+#@title Please run this cell first!
```

Comment:
Please add a module-level docstring explaining the purpose of this module and its contents. Also, please add a license header.

Reply:
Module-level docstring and license header will be added, thank you.